apps/condo/domains/common/components/Table/Renders.tsx

```diff
@@ -89,79 +90,62 @@ export const renderMeterReading = (values: string[], resourceId: string, measure
     return `${getIntegerPartOfReading(values[0])} ${measure}`
 }
 
-const fillMoneyWithSpaces = (substring: string, startIndex: number, spaces: Array<number>) => {
-    const chars = []
-    for (let i = 0; i < substring.length; i++) {
-        const absoluteIndex = startIndex + i
-        const char = substring.charAt(i)
-        if (spaces.includes(absoluteIndex)) {
-            chars.push(MONEY_PARTS_SEPARATOR)
+const getCurrencySymbol = (currencyCode) => {
+    return (0).toLocaleString(
+        undefined,
+        {
+            style: 'currency',
+            currency: currencyCode,
+            minimumFractionDigits: 0,
+            maximumFractionDigits: 0,
+            currencyDisplay: 'narrowSymbol',
```

Comment:
What's this? sounds like custom logic...

Reply:
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormatNo, this is Intl format logic